server/http_status.go

```diff
@@ -257,11 +257,14 @@ func (s *Server) startHTTPServer() {
 	fetcher := sqlInfoFetcher{store: tikvHandlerTool.Store}
 	serverMux.HandleFunc("/debug/sub-optimal-plan", fetcher.zipInfoForSQL)
 
+	// failpoint is enabled only for tests so we can add some http APIs here for tests.
 	failpoint.Inject("integrateFailpoint", func() {
```

Comment:
Do we need to rename this failpoint (eg `integrateTestAPIs`) since it inject multiple handlers now?

Reply:
Yes! I was going to rename it but I'm afraid it will affect some scripts so I kept it.

Reply:
There are only a few scripts and I can update them quickly, so feel free to rename the failpoint.

Reply:
I can't think of a good name. I rename it to `enableTestAPI`. What do you think?